documentation/user_docs/docs/manual/Multifit.rst

```diff
@@ -2,146 +2,185 @@
+By default, multifit will allow all these parameters to float freely in the fit. However, suppose you want to keep the
+Gaussian centred on the origin. Then you can provide a list of which parameters are allowed to float (1) or are fixed
+(0). In this example:
 
 ::
 
-   >> kk = kk.set_free ([1,0,1]); \t% keep the second parameter fixed in the fit
+   >> kk = kk.set_free([1, 0, 1]); % keep the second parameter fixed in the fit
 
 At this point, you can perform a fit:
 
 ::
 
-   >> [my_fitted_data, fit_params] = kk.fit;
+   >> [my_fitted_data, fit_params] = kk.fit();
 
 
-This returns two arguments: ``my_fitted_data`` is an array of objects which is the same as the input data except that the signal (or equivalently, the intensity) is set to the values calculated at the fitted parameter values; and ``fit_params``, which is a structure that contains the fitted parameter values, estimated errors on those fitted values, the value of chi-squared for the fit and the covariance matrix of the fitted parameters.
+This returns two arguments: ``my_fitted_data`` is an array of objects which is the same as the input data except that
+the signal (or equivalently, the intensity) is set to the values calculated at the fitted parameter values; and
+``fit_params``, which is a structure that contains the fitted parameter values, estimated errors on those fitted values,
+the value of chi-squared for the fit and the covariance matrix of the fitted parameters.
 
-If you want to see how the fit is progressing from one iteration to the next, and also get a listing in the Matlab command window of the final fit parameters, you can ask for more verbose output by changing one of the multifit options:
+If you want to see how the fit is progressing from one iteration to the next, and also get a listing in the Matlab
+command window of the final fit parameters, you can ask for more verbose output by changing one of the multifit options:
 
 ::
 
-   >> kk = kk.set_options ('list',2); \t% prints highly verbose output to the screen
+   >> kk = kk.set_options('list', 2); % prints highly verbose output to the screen
 
 
-Other options change the fit convergence criteria, and whether or not the final fit is calculated only are data points that remained once points with zero error bars were removed or at all data points.
+Other options change the fit convergence criteria, and whether or not the final fit is calculated only are data points
+that remained once points with zero error bars were removed or at all data points.
 
-Fitting can be computationally very expensive. Before you start fitting, it can be very useful to simulate at the initial parameters to see if your starting point is reasonable:
+Fitting can be computationally very expensive. Before you start fitting, it can be very useful to simulate at the
+initial parameters to see if your starting point is reasonable:
```

Comment:
could we define reasonable, e.g. calculated values visually similar to the experimental ones